test_scripts/Defects/6_1/1918_Multiple_apps_activation_during_active_embedded_audio_source.lua

```diff
@@ -0,0 +1,145 @@
+      config["application"..pAppId].registerAppInterfaceParams.appHMIType[1] == "DEFAULT"
+      then
+        return data.payload.audioStreamingState == "NOT_AUDIBLE" and
+          data.payload.hmiLevel == "FULL" and
+          data.payload.systemContext == "MAIN"
+      else
+        return data.payload.audioStreamingState == "AUDIBLE" and
+          data.payload.hmiLevel == "FULL" and
+          data.payload.systemContext == "MAIN"
+    end
+  end)
+end
+
+local function deactivateApp(pAppId)
+  if not pAppId then pAppId = 1 end
+  common.getHMIConnection():SendNotification("BasicCommunication.OnAppDeactivated", { appID = common.getHMIAppId(pAppId) })
+  common.getMobileSession(pAppId):ExpectNotification("OnHMIStatus")
+  :ValidIf(function(_, data)
+    if config["application"..pAppId].registerAppInterfaceParams.isMediaApplication == true
+      then
+        return data.payload.audioStreamingState == "AUDIBLE" and
+          data.payload.videoStreamingState == "NOT_STREAMABLE" and
+          data.payload.hmiLevel == "LIMITED" and
+          data.payload.systemContext == "MAIN"
+      else
+        return data.payload.audioStreamingState == "NOT_AUDIBLE" and
+          data.payload.videoStreamingState == "NOT_STREAMABLE" and
+          data.payload.hmiLevel == "BACKGROUND" and
+          data.payload.systemContext == "MAIN"
+    end
+  end)
+end
+
+local function embededAudioActivated()
+  common.getHMIConnection():SendNotification("BasicCommunication.OnEventChanged",
+    { eventName = "AUDIO_SOURCE", isActive = true })
+  common.getMobileSession(1):ExpectNotification("OnHMIStatus", {
+    hmiLevel = "BACKGROUND", audioStreamingState = "NOT_AUDIBLE", videoStreamingState = "NOT_STREAMABLE"
+  })
+  common.getMobileSession(2):ExpectNotification("OnHMIStatus", {
+    hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", videoStreamingState = "NOT_STREAMABLE"
+  })
+end
+
+local function activateNaviApp()
+  local requestId = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId(2) })
+  EXPECT_HMIRESPONSE(requestId)
+  common.getMobileSession(2):ExpectNotification("OnHMIStatus",
+    {hmiLevel = "FULL", audioStreamingState = "NOT_AUDIBLE" })
+end
+
+local function activateMediaApp()
+  common.getHMIConnection():SendNotification("BasicCommunication.OnEventChanged",
+    { eventName = "AUDIO_SOURCE", isActive = false })
+  common.getMobileSession(1):ExpectNotification("OnHMIStatus",
+    { hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE" },
+    { hmiLevel = "FULL", audioStreamingState = "AUDIBLE" })
+  :Times(2)
+  :Do(function(exp)
+    if exp.occurences == 1 then
+      local requestId = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId(1) })
+      EXPECT_HMIRESPONSE(requestId)
+    end
+  end)
+  common.getMobileSession(2):ExpectNotification("OnHMIStatus",
+    { hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE" })
```

Comment:
```suggestion
  -- hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", videoStreamingState = "STREAMABLE"
  common.getMobileSession(2):ExpectNotification("OnHMIStatus"):Times(0)
```
App would be in `LIMITED`, `AUDIBLE`, `STREAMABLE`